process/smartContract/hooks/blockChainHook.go

```diff
@@ -771,6 +775,24 @@ func (bh *BlockChainHookImpl) EpochConfirmed(epoch uint32, _ uint64) {
 	}
 }
 
+// ExecuteSmartContractCallOnOtherVM on another VM
+func (bh *BlockChainHookImpl) ExecuteSmartContractCallOnOtherVM(input *vmcommon.ContractCallInput) (*vmcommon.VMOutput, error) {
```

Comment:
how can we prevent this call being made only on some hand-picked setups? Something like when we instantiate wasm-vms, we tell them: I allow/disallow you to use the API called `ExecuteSmartContractCallOnOtherVM`
Reason: I want to specifically tell the VM instantiations in the elrond-go repo *not* to allow the usage of this API. 
Otherwise, we need to have the SetVMContainer function declared in another component that contains the word `sovereign` in its naming. The reason is again, prevention of usage in elrond-go setup.

Reply:
We should call the SetVMContainer with empty container - or with not container at all ?